compiler_opt/rl/inlining/gin_configs/common.gin

```diff
@@ -7,3 +7,9 @@ clang_path=None
 runners.InliningRunner.llvm_size_path=%llvm_size_path
 runners.InliningRunner.clang_path=%clang_path
 runners.InliningRunner.launcher_path=%launcher_path
+
+problem_config.flags_to_add.add_flags=()
+problem_config.flags_to_delete.delete_flags=('-split-dwarf-file','-split-dwarf-output','-fprofile-sample-use','-fprofile-remapping-file',)
```

Comment:
please don't delete -fprofile-sample-use and -fprofile-remapping-file`, if present, they *must* be kept (and remapped). Otherwise compilation succeeds but the result is not what we want - i.e. a nasty bug to identify.

Reply:
To confirm, is it that these flags only appear in AFDO case, which should be a replace; and they should not exist in the FDO case, so should not be deleted as well?

so FDO:
delete flags = -split-dwarf-file, -split-dwarf-output
replace = {}

AFDO:
delete flags = -split-dwarf-file, -split-dwarf-output
replace = {'-fprofile-sample-use','-fprofile-remapping-file'}

Reply:
ahh, makes sense. i updated the delete_flags list https://github.com/google/ml-compiler-opt/pull/103/commits/d0cbcdf7906cc1c90e2971cd9a43f538a359cf0b

Reply:
@yundiqian that's correct, yes.